File: javascript/conditions/bike.js
a = parseInt(prompt('Enter a number '));
if (a>100000) {
    tax=0.15*a;
    console.log(`${a}Rupees,tax=${tax}`);
}
else if (a>50000 && a<=100000) {
    tax=0.10*a;
    console.log(`${a}Rupees,tax=${tax}`);
}
else if (a<=50000) {
    tax=0.05*a;
    console.log(`${a}Rupees,tax=${tax}`);
}
else {
    console.log("Invalid input");
}

File: python/work/bike.py
a=int(input("Enter your bike price:"))
if(a>100000):
    tax=0.15*a
    print(a,"Rupees","tax=",tax)
elif(a>50000 and a<=100000):
    tax=0.10*a
    print(a,"Rupees","tax=",tax)
elif(a<=50000):
    tax=0.05*a
    print(a,"Rupees","tax=",tax)
else:
    print("invalid input")